support-frontend/app/views/JsStringLiteral.scala

```diff
@@ -0,0 +1,25 @@
+package views
+
+import play.twirl.api.Html
+
+object JsStringLiteral {
+
+  def apply(rawString: String): Html = {
```

Comment:
after moving the escaper over, I realised it wasn't going to be as safe as I'd hoped.
This implementation should handle special characters both in Html and Javascript.
The mappings I got from https://stackoverflow.com/questions/8749001/escaping-html-entities-in-javascript-string-literals-within-the-script-block
I have tested it and it seems to work
string `TEST-CODE < > " ' \ & `
ended up in the source as:
![image](https://user-images.githubusercontent.com/7304387/85306525-83fa2100-b4a6-11ea-8a42-bc4e06ed52fc.png)
and in the page showed up as 
![image](https://user-images.githubusercontent.com/7304387/85306604-a0965900-b4a6-11ea-815b-17b75d77e337.png)